src/esp/gfx/PbrShader.h

```diff
@@ -172,41 +164,102 @@ class PbrShader : public Magnum::GL::AbstractShaderProgram {
      * see PBR fragment shader code for more details
      * Requires the @ref Tangent4 attribute to be present.
      */
-    PrecomputedTangent = 1 << 10,
+    PrecomputedTangent = 1 << 9,
 
     /**
      * Enable object ID output for this shader.
      */
-    ObjectId = 1 << 11,
+    ObjectId = 1 << 10,
 
     /**
      * Support Instanced object ID. Retrieves a per-instance / per-vertex
      * object ID from the @ref ObjectId attribute. If this is false, the shader
      * will use the node's semantic ID
      */
-    InstancedObjectId = (1 << 12) | ObjectId,
+    InstancedObjectId = (1 << 11) | ObjectId,
+
+    /**
+     * Has ClearCoat layer.
+     */
+    ClearCoatLayer = 1 << 12,
+    /**
+     * Has ClearCoat Texture in ClearCoat layer
+     */
+    ClearCoatTexture = (1 << 13) | ClearCoatLayer,
```

Comment:
I haven't seen this convention before so maybe a comment explaining. I guess you are basically grouping similar enum values to have a shared bit.

Reply:
Also, I see we're using these with `Corrade::EnumSet` and from [here](https://doc.magnum.graphics/corrade/classCorrade_1_1Containers_1_1EnumSet.html) I see "The only requirement for the enum type is that all the values must be binary exclusive." @mosra thoughts?

Reply:
I'll remove that line from the docs. Having a particular flag a superset of another is a very common use case, so the "requirement" makes no sense nowadays.